module4/ex16_magic_tuples/ex16_magic_tuples.py

```diff
@@ -1,5 +1,6 @@
-def magic_tuples():
-    pass
+def magic_tuples(total, max_value):
+    for i in range(total - max_value + 1, max_value):
```

Comment:
What if `max_value` is greater than `total`? For example, `magic_tuples(5, 8)`.

(I will make sure to add an extra test for this edge case.)